client/src/panels/panel_declarations/finances/auth_exp_planned_spending/auth_exp_planned_spending.js

```diff
@@ -273,10 +273,38 @@ class AuthExpPlannedSpendingGraph extends React.Component {
 }
 
 const render = function ({ calculations, footnotes, sources, glossary_keys }) {
-  const { info, panel_args, subject } = calculations;
+  const { panel_args, subject } = calculations;
   const { data_series, additional_info } = panel_args;
 
-  const final_info = { ...info, ...additional_info };
+  const array_avg = _.chain(data_series)
+    .map((type) => [
+      type.key,
+      _.reduce(
+        type.values,
+        (result, year_spending) => result + year_spending,
+        0
+      ) / type.values.length,
+    ])
+    .fromPairs()
+    .value();
+
+  const planned_spending = _.chain(data_series)
```

Comment:
No need to filter this, take a look at line 375. Just do straight `spending[length-1]` there